driver_cpl/driver/cesm_comp_mod.F90

```diff
@@ -1072,25 +1095,95 @@ subroutine cesm_init()
    call component_init_pre(ice, ICEID, CPLICEID, CPLALLICEID, infodata, ntype='ice')
    call component_init_pre(glc, GLCID, CPLGLCID, CPLALLGLCID, infodata, ntype='glc')
    call component_init_pre(wav, WAVID, CPLWAVID, CPLALLWAVID, infodata, ntype='wav')
+   call t_stopf('comp_init_pre_all')
 
 #ifdef ESMF_INTERFACE
+   call t_startf('comp_init_cc_atm')
```

Comment:
these timer start and stops should be inside the component_init_cc subroutine.   No need to bring them out.